.circleci/config.yml

```diff
@@ -32,6 +32,62 @@ jobs:
           name: website deploy
           command: ./scripts/deploy.sh
 
+  # build frontend yarn cache
+  frontend-cache:
+    docker:
+      - image: circleci/node:8
+    steps:
+      - checkout
+      
+      # cache yarn deps
+      - restore_cache:
+          key: consul-ui-v1-{{ checksum "ui-v2/yarn.lock" }}
```

Comment:
Tiny thing but, should this read:

```
key: consul-ui-v2-{{ checksum "ui-v2/yarn.lock" }}
```
Note the `v2` in the key not `v1`

?

Reply:
The key here is a unique key that CircleCI uses to save caches, the v1 is a bit confusing here because we are looking at the ui-v2 folder. As suggested in the [circleci docs](https://circleci.com/docs/2.0/caching/#clearing-cache), if you want to regenerate/clear a cache you should bump a version and build a new one. In this case I wanted v1 to refer to the first version of this cache and not the v1 UI. 

As an example use case if the checksum was just `12345` the first cache would be saved as `consul-ui-v1-12345` but if for any reason that generated a bad cache or we wanted to make a new one, we could bump it to `consul-ui-v2-12345` (when we just want a new cache without changing anything in the lock file).

Reply:
Ah cool gotcha, yeah sounds good to me, lets keep the `consul-ui-v2-12345` cachebusting.

It's just unfortunate right now that we have a version in the folder name, I think your suggestion of the extra cache busting number on the end here (and keeping v2 always) means it's less like to confuse.

👍